src/sid/parse_model.py

```diff
@@ -175,7 +175,11 @@ def parse_virus_strains(
         if len(virus_strains) == 0:
             raise ValueError("The list of 'virus_strains' cannot be empty.")
 
-        expected_indices = [("virus_strain", name, "factor") for name in virus_strains]
+        expected_indices = [
+            ("virus_strain", name, factor)
+            for name in virus_strains
+            for factor in ("contagiousness_factor", "immunity_resistance_factor")
+        ]
```

Comment:
``"factor"`` is renamed to ``"contagiousness_factor"`` to avoid confusion with the new factor ``"immunity_resistance_factor"``.